src/transformation/visitors/optional-chaining.ts

```diff
@@ -90,16 +97,16 @@ export function transformOptionalChain(context: TransformationContext, node: ts.
 
 export function transformOptionalChainWithCapture(
     context: TransformationContext,
-    node: ts.OptionalChain,
+    tsNode: ts.OptionalChain,
     thisValueCapture: lua.Identifier | undefined,
     isDelete?: ts.DeleteExpression
 ): ExpressionWithThisValue {
-    const luaTemp = context.createTempNameForNode(node);
+    const luaTempName = context.createTempName("opt");
```

Comment:
Why change this from what it was?

Reply:
"node" was too generic and so confusing; there are a lot of variables, TS and Lua, in the body.

Reply:
It wasn't actually "node" literal though, it was already deducing a very specific name based on what the node is, see the changes in the snapshots where it changed `____table_has_result_0` to `___opt_0`

Reply:
Sorry, earlier I thought the comment was addressing something else.

The reason for this is that long-ish optional chains, which are somewhat common, generate long names such as "____a_b_c_foo_result_c_1". "____opt" is shorter and still signifies what the original TS was.